.golangci.yml

```diff
@@ -1,231 +1,247 @@
-
-    # Report preallocation suggestions only on simple loops that have no returns/breaks/continues/gotos in them.
-    # True by default.
-    simple: true
-    range-loops: true # Report preallocation suggestions on range loops, true by default
-    for-loops: false # Report preallocation suggestions on for loops, false by default
-  gocritic:
-    # Which checks should be enabled; can't be combined with 'disabled-checks';
-    # See https://go-critic.github.io/overview#checks-overview
-    # To check which checks are enabled run `GL_DEBUG=gocritic golangci-lint run`
-    # By default list of stable checks is used.
-    enabled-checks:
-
-    # Which checks should be disabled; can't be combined with 'enabled-checks'; default is empty
-    disabled-checks:
-      - regexpMust
-      - rangeValCopy
-      - hugeParam
-      - ifElseChain
-      - singleCaseSwitch
-
-    # Enable multiple checks by tags, run `GL_DEBUG=gocritic golangci-lint run` to see all tags and checks.
-    # Empty list by default. See https://github.com/go-critic/go-critic#usage -> section "Tags".
-    enabled-tags:
-      - performance
-
-    settings: # settings passed to gocritic
-      captLocal: # must be valid enabled check name
-        paramsOnly: true
-  godox:
-    # report any comments starting with keywords, this is useful for TODO or FIXME comments that
-    # might be left in the code accidentally and should be resolved before merging
-    keywords: # default keywords are TODO, BUG, and FIXME, these can be overwritten by this setting
-      - NOTE
-      - OPTIMIZE # marks code that should be optimized before merging
-      - HACK # marks hack-arounds that should be removed before merging
-  dogsled:
-    # checks assignments with too many blank identifiers; default is 2
-    max-blank-identifiers: 2
-
-  whitespace:
-    multi-if: false   # Enforces newlines (or comments) after every multi-line if statement
-    multi-func: false # Enforces newlines (or comments) after every multi-line function signature
-
-  wsl:
-    # If true append is only allowed to be cuddled if appending value is
-    # matching variables, fields or types on line above. Default is true.
-    strict-append: true
-    # Allow calls and assignments to be cuddled as long as the lines have any
-    # matching variables, fields or types. Default is true.
-    allow-assign-and-call: true
-    # Allow multiline assignments to be cuddled. Default is true.
-    allow-multiline-assign: true
-    # Allow case blocks to end with a whitespace.
-    allow-case-traling-whitespace: true
-    # Allow declarations (var) to be cuddled.
-    allow-cuddle-declarations: false
-
-linters:
-  fast: false
-  enable-all: true
-  disable:
-    # TODO Enforce the below linters later
-    - dupl
-    - errcheck
-    - funlen
-    - gocyclo
-    - gocognit
-    - lll
-    - scopelint
-    - unparam
-    - wsl
-    - gomnd
-    - godot
-    - testpackage
-    - goerr113
-    - gci
+# options for analysis running
```

Comment:
Did you change all the line endings in the file as well? For future reference, it's helpful to separate widespread formatting changes from code changes into separate commits so that the diff tool can highlight the more important code changes in one of the commits.

Reply:
I use `dos2unix` :)

Reply:
Good to know! So the advice is that next time, your should do a commit after running `dos2unix` on the file, then do your code changes in a separate commit so that a reviewer can inspect the code changes independent from the formatting changes in the GitHub PR view.

Reply:
Sure.